src/components/Error.tsx

```diff
@@ -0,0 +1,35 @@
+/* eslint-disable jsx-a11y/control-has-associated-label */
+
+import { useEffect, useState } from 'react';
+
+interface ErrorProps {
+  error:string,
+}
+
+export const Error: React.FC<ErrorProps> = ({ error }) => {
+  const [message, setMessage] = useState(error);
+  const [isHidden, setIsHidden] = useState(false);
+
+  useEffect(() => {
+    setMessage(error);
+    if (error.length) {
+      setIsHidden(false);
```

Comment:
you should just set this onClick, and you don't need a state for `message` and useEffect here

Reply:
> you should just set this onClick, and you don't need a state for message and don'e use useEffect here